src/intervals/interval_operations/constants.jl

```diff
@@ -16,15 +16,12 @@ argument given, the default interval bound type is used.
 
 Implement the `empty` function of the IEEE Std 1788-2015 (section 10.5.2).
 """
-emptyinterval(::Type{F}) where {F<:Interval} = F(Inf, -Inf)
-emptyinterval(::F) where {F<:Interval} = emptyinterval(F)
-emptyinterval(::Type{Complex{F}}) where {F<:Interval} = complex(emptyinterval(F), emptyinterval(F))
-
-# NOTE Here the restriction should be any allowed bound type. Using
-# AbstractFloat for simplicity for now
-emptyinterval(::Type{T}) where {T<:AbstractFloat} = emptyinterval(Interval{T})
+emptyinterval(::Type{F}) where {T<:NumTypes,F<:Interval{T}} = F(typemax(T), typemin(T))
```

Comment:
Note that
```julia
julia> emptyinterval(Interval{Rational{Int}})
ERROR: MethodError: no method matching Rational{Int64}(::Float64, ::RoundingMode{:Down})
...
```

I think we need to define a specialized method for rational intervals, perhaps
```julia
emptyinterval(::Type{F}) where {T<:Integer,F<:Interval{Rational{T}}} = F(typemax(T)//one(T), typemin(T)//one(T))
```

Reply:
That is strange, I cannot reproduce this error (on Julia 1.9-rc2).

Reply:
You are right... I think I was checking a different branch... sorry.